drivers/ethernet/dwc_xgmac/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
```

Comment:
why create a new library?

Reply:
If we don't add this line we will get below build error 

```
CMake Error at ../../../../cmake/modules/extensions.cmake:425 (add_library):
  No SOURCES given to target: mbedTLSBase
Call Stack (most recent call first):
  ../../../../modules/mbedtls/CMakeLists.txt:33 (zephyr_library_named)
```

Reply:
removed